src/mscorlib/src/System/Span.cs

```diff
@@ -86,13 +86,18 @@ private Span(ref T ptr, int length)
 
         public int Length
         {
-            get
-            {
-                return _length;
-            }
+            get { return _length; }
+        }
+
+        internal uint SizeInBytes
```

Comment:
Using `uint` for sizes is problematic because it can overflow on 64-bit platforms. You may want to use `nuint` like here: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Buffer.cs#L18